crates/hyperswitch_interfaces/src/lib.rs

```diff
@@ -1,7 +1,17 @@
 //! Hyperswitch interface
 
-#![warn(missing_docs, missing_debug_implementations)]
```

Comment:
Can we add this back and address warnings instead of removing the lint altogether?

Reply:
Done

Reply:
For future reference: #5011 was opened for adding better documentation comments, we'll be taking it up separately.